js/packages/cli/src/candy-machine-v2-cli.ts

```diff
@@ -1002,16 +1002,27 @@ programCommand('mint_multiple_tokens')
     log.info(`Minting ${NUMBER_OF_NFTS_TO_MINT} tokens...`);
 
     const mintToken = async index => {
+		  var max = 30; //seconds
+		  var min = 1; //seconds
+		  var rand = Math.floor(Math.random() * (max - min + 1) + min); //Generate Random number between 5 - 10
+		  log.info(`waiting ${rand} seconds`)
+		  await new Promise(f => setTimeout(f, rand * 1000)); // Wait random seconds to not get 429 too many requests
```

Comment:
429 should not be the case for private RPC, I don't think it makes sense to handle this, you also want to make the batch size configurable